app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -131,6 +131,26 @@ data class AppSettings(
         defaultValue = "1",
     )
     val backConfirmationMode: Int,
+    @ColumnInfo(
+        name = "comment_sorting_mode",
+        defaultValue = "0",
+    )
```

Comment:
Whoa, probably should asked about this one before going ahead with it.

- These should be stored in lemmy's back end, otherwise no other app / front end will be able to use them. 
  - Same goes for default sorts for specific communities, which might also need back-end settings.
- So now we have sorts stored in 3 (maybe 4) different places: 
  - Your lemmy backend profile
  - The jerboa `Account` table (which is where these should live, for fast initial app loading)
  - And now also the jerboa `AppDB` table for some reason, which overrides each account?

Reply:
I did ask about this in matrix chat. If these are stored on the backend, we would need to add them first there. And then we would also have the compatibility issue again that older lemmy servers won't have these. Further anonymous account wont be able to use these either. This would only add the sorting options that don't exist yet. I have also been contemplating to make the anon account, an actual "special" account that is stored in DB. This would allow us to have anon all share the features of a logged in account. All global settings now can be saved per account based, such as the "censor NSFW".  This would a huge migration but better earlier than later.